python/ray/tests/kuberay/test_autoscaling_config.py

```diff
@@ -312,6 +330,14 @@ def test_resource_quantity(input: str, output: int):
             None,
             id="autoscaler-options",
         ),
+        pytest.param(
+            _get_ray_cr_with_groups_suspended(),
+            _get_autoscaling_config_with_groups_suspended(),
```

Comment:
This tests both max_workers and min_workers are set to 0 in the generated autoscaling config.